src/Application/Migrations/22_view_serviceable_tickets.sql

```diff
@@ -36,7 +36,7 @@ CREATE OR REPLACE VIEW view_serviceable_tickets AS
 
 	WHERE
 		pj.read_only = false AND
-		t.ticket_type != 'meta' AND
+		t.ticket_type IN ('recording','encoding','ingest') AND
```

Comment:
Because for some reasons Postgres doesnt get it that this is actually the same. The newer line will result in using the ANY operation which will in turn make use of a hash index, while using != will result in a "simple filter" on a full index scan or even table scan.